src/backend/distributed/utils/citus_stat_tenants.c

```diff
@@ -683,7 +740,8 @@ FindTenantStats(MultiTenantMonitor *monitor)
 	for (int i = 0; i < monitor->tenantCount; i++)
 	{
 		TenantStats *tenantStats = &monitor->tenants[i];
-		if (strcmp(tenantStats->tenantAttribute, AttributeToTenant) == 0 &&
+
+		if (strncmp(tenantStats->tenantAttribute, AttributeToTenant, MAX_TENANT_ATTRIBUTE_LENGTH) == 0 &&
```

Comment:
Why was this change necessary?

Reply:
I don't have a solid reason. I was just curious about its impact on performance. In any case, it's no longer necessary.